### PR TITLE
Update r.rb formula for R version 3.1.3

### PR DESCRIPTION
**Pull request by amoeba**
Updates r.rb to the latest version of R.

R 3.1.3 was released on March 9th. See the mailing list announcement for more information: http://article.gmane.org/gmane.comp.lang.r.announce/157

### CODE REVIEW

File: r.rb [hunk 1]
```diff
@@ -7,9 +7,9 @@ def stage
 
 class R < Formula
   homepage "http://www.r-project.org/"
-  url "http://cran.rstudio.com/src/base/R-3/R-3.1.2.tar.gz"
-  mirror "http://cran.r-project.org/src/base/R-3/R-3.1.2.tar.gz"
-  sha256 "bcd150afcae0e02f6efb5f35a6ab72432be82e849ec52ce0bb89d8c342a8fa7a"
+  url "http://cran.rstudio.com/src/base/R-3/R-3.1.3.tar.gz"
+  mirror "http://cran.r-project.org/src/base/R-3/R-3.1.3.tar.gz"
+  sha256 "07e98323935baa38079204bfb9414a029704bb9c0ca5ab317020ae521a377312"
   revision 1
```
Comment (iml) [inline r.rb]: Please remove the revision.

**Comment by amoeba** [comment]
Revision removed as per request of iml. Looks like it was you (iml) that added the revision to the formula? https://github.com/Homebrew/homebrew-science/commit/a9669cfd05129e9335b64c089d3bd3cd542ef390

**Comment by iml** [comment]
Yes the revision forces an upgrade, such as when a dependency upgrade breaks things, but needs to be removed when there's a new version.

**Comment by amoeba** [comment]
Gotcha. That makes sense. Read this too https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#formulae-revisions

Learn something new *and* keep your software up to date.

Thanks!

**Pull request closed without merging**